sources/vcdMerge/src/Merge.cpp

```diff
@@ -60,62 +61,72 @@ void MERGE::Merge::Run()
     // Create the output signal database and set its base time unit.
     m_pMerged = std::make_unique<SIGNAL::SignalDb>(m_TimeUnit);
 
-    // Find the longest leading time for among all sources.
-    m_MaxLeadingTime = FindMaxLeadingTime();
+    // Find the longest leading time among all sources.
+    try
+    {
+        m_MaxLeadingTime = FindMaxLeadingTime();
+    }
+    catch (std::runtime_error &)
```

Comment:
This is very unlikely to happen. Still an exception. It is not visible to the end user as warning nor error and is handled internally in the upper layer.